test/unit/com/unifina/service/ClusterServiceSpec.groovy

```diff
@@ -204,7 +204,7 @@ class ClusterServiceSpec extends Specification {
 		service.canvasService = Mock(CanvasService)
 		CanvasesPerNode canvasesPerNode = new CanvasesPerNode()
 		canvasesPerNode.shouldBeRunning = new ArrayList<HashMap<String, Object>>()
-		SecUser u = new SecUser(username: "pena@host.com", password: "abcabcabcabc123", name: "Pena", timezone: "Europe/Helsinki")
+		SecUser u = new SecUser(username: "pena@host.com", password: "abcabcabcabc123", name: "Pena", lastLogin: new Date())
```

Comment:
Can remove `lastLogin` if you set the default field value in `SecUser.groovy` (also elsewhere in this file)